### PR TITLE
Undo fixed

### PR DESCRIPTION
**Pull request by Su1eym4n**
Now user can undo drag and creation of the object 
Flaws:
Ctrl-Z works only when mouse is focuses on section or page (not sure why)

### CODE REVIEW

File: modules/undo.py [hunk 1]
```diff
@@ -1,13 +1,20 @@
+
```
Comment (rockCityMath) [inline modules/undo.py:1]: I think we're trying to just use PySide imports, can you make these pyside for consistency?

File: modules/page.py [hunk 2]
```diff
@@ -24,8 +24,8 @@ def add_page(editor):
         add_section(editor)
         editor.autosaver.onChangeMade()
         
-        cmd = Undo({'type':'page','name':title, 'action':'create'})
-        editor.undo_stack.append(cmd)
+        # cmd = Undo({'type':'page','name':title, 'action':'create'})
```
Comment (rockCityMath) [inline modules/page.py:27]: can u take these commented lines here and in the other places out if we dont need them

File: modules/undo.py [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
-    def redo(self):
-        # Implement the operation that this command represents
-        print(self.parameter+'redo')
 
-    def undo(self):
+    def undo(self, editor):
         # Implement the undo operation for this command
-        print(self.parameter+'undo')
+        if self.parameter['action']=='move':
+            print('move obj to old location')
```
Comment (rockCityMath) [inline modules/undo.py:15]: can u take out the print statements if ur done with the debugging

File: screens/editor.py [hunk 4]
```diff
@@ -46,8 +53,46 @@ def dropEvent(self, event):
         self.focusWidget().move(event.pos().x(), event.pos().y())
         event.acceptProposedAction()
         self.autosaver.onChangeMade()
-
+        self.undo_stack += self.temp_buffer[:1]
+        self.temp_buffer = []
+        
+        # debugginh
```
Comment (rockCityMath) [inline screens/editor.py:59]: can u take this out

**Review by rockCityMath — CHANGES_REQUESTED** [review]

File: modules/undo.py (hunk 1 above)
Comment (Su1eym4n) [reply modules/undo.py]: Done

File: modules/page.py (hunk 2 above)
Comment (Su1eym4n) [reply modules/page.py]: Done

File: modules/undo.py (hunk 3 above)
Comment (Su1eym4n) [reply modules/undo.py]: Done

**Review by rockCityMath — APPROVED** [review]

**Pull request merged**